src/Microsoft.Health.Fhir.SpecManager/Models/FhirElement.cs

```diff
@@ -141,7 +140,7 @@ public class FhirElement : FhirTypeBase
             }
 
             IsModifier = isModifier == true;
-            IsSummary = (isSummary == true) || IsModifier;
+            IsSummary = isSummary == true;
```

Comment:
I agree with this change: although it seems logical that modifier elements are also in summary, the spec itselfs needs to enforce this if this is indeed true.  We should just reflect the spec here.  Note that this is shared code with the other code generators, so this might well have regressions.

Reply:
`IsModifier` is now an attribute of FhirElement. The determination whether an element should be in summary or not, is left to the SDK. So the logic is shifted from here to the SDK.